src/build/app/mod.rs

```diff
@@ -1517,34 +1537,6 @@ impl<'b> App<'b> {
     }
 
     pub fn _propagate(&mut self, prop: Propagation) {
-        macro_rules! propagate_subcmd {
```

Comment:
Why did you moved this macro to `macros.rs`? It doesn't look like it's used outside of this file at all